tests/functional/z_cluster/test_ceph_default_values_check.py

```diff
@@ -129,12 +129,26 @@ def test_validate_ceph_config_values_in_rook_config_override(self):
             stored_values = constants.ROOK_CEPH_CONFIG_VALUES_412.split("\n")
         elif ocs_version == version.VERSION_4_14 or ocs_version == version.VERSION_4_15:
             stored_values = constants.ROOK_CEPH_CONFIG_VALUES_414.split("\n")
-        elif ocs_version >= version.VERSION_4_16:
+        elif ocs_version in [
+            version.VERSION_4_16,
+            version.VERSION_4_17,
+            version.VERSION_4_18,
+        ]:
             stored_values = constants.ROOK_CEPH_CONFIG_VALUES_416.split("\n")
+        elif ocs_version == version.VERSION_4_19:
+            stored_values = constants.ROOK_CEPH_CONFIG_VALUES_419.split("\n")
         else:
             stored_values = constants.ROOK_CEPH_CONFIG_VALUES.split("\n")
         log.info(f"OCS version is {ocs_version}")
         log.info(f"Stored values are {stored_values}")
+        # After all other filtering, normalize empty lines by replacing all empty lines with a fixed count
+        num_empty_lines = 0  # Replace with 0 to remove all empty lines or any other number to keep a fixed count
+        config_data = [line for line in config_data if line != ""] + [
+            ""
+        ] * num_empty_lines
+        stored_values = [line for line in stored_values if line != ""] + [
+            ""
+        ] * num_empty_lines
```

Comment:
If `num_empty_lines = 0`, we can just write: 

> config_data = [line for line in config_data if line != ""]
stored_values = [line for line in stored_values if line != ""]